Shifty.App/Shared/NavMenu.razor

```diff
@@ -3,12 +3,16 @@
 
 <MudNavMenu>
     <MudNavLink Href="" Match="NavLinkMatch.All" Icon="@Icons.Material.Filled.Home">Home</MudNavLink>
-    @if (_user is not null && _user.IsInRole("Board"))
+    @if (_user is not null && _user.IsInRole("Board") || _user is not null && _user.IsInRole("Manager"))
```

Comment:
WHy check if the user is null twice? 


Also, let's look into authorizedView for this as well

Reply:
I have removed the duplicate null check. See the previous comment about the authorizedview.